app/components/post_list/post_list.js

```diff
@@ -187,7 +185,7 @@ export default class PostList extends PureComponent {
                 keyExtractor={this.keyExtractor}
                 ListFooterComponent={this.renderChannelIntro}
                 onEndReached={this.loadMorePosts}
-                onEndReachedThreshold={700}
+                onEndReachedThreshold={0}
```

Comment:
Does this not cause the post list to pause when it's loading more? It can be less than 700, but maybe not right to 0

Reply:
I got this value after reading the issues in the RN repo, if we leave it as it was we will get a `onEndReached` being trigger unnecessarily